elasticapm/instrumentation/packages/dbapi2.py

```diff
@@ -170,10 +175,17 @@ def executemany(self, sql, param_list):
         return self._trace_sql(self.__wrapped__.executemany, sql,
                                param_list)
 
+    def _bake_sql(self, sql):
+        """
+        Method to turn the "sql" argument into a string
+        """
+        return sql
```

Comment:
The method description might be a bit confusing, as it actually does nothing in this class.
Would it do any harm if you'd actually move the `as_string` check here?

Reply:
the `as_string` check is specific to psycopg2 (the Python PostgreSQL driver), so I think putting it in the psycopg2-specific subclass makes sense. However I agree that the docstring needs work.